pkg/config/config.go

```diff
@@ -88,6 +88,10 @@ var CommunicationConfigFileName = "comm_config.yaml"
 // Notify flag to toggle event notification
 var Notify = true
 
+//KubeCtlLinkedChannels is a set that contains the channel ID (as a key)of all channels linked to
+//this cluster and containing the default namespace as a value
```

Comment:
```suggestion
// KubeCtlLinkedChannels is a set that contains the channel ID (as a key)of all channels linked to
// this cluster and containing the default namespace as a value
```
Please add leading space in comments to have consistent pattern across the source code